protobuf/ns/ns.proto

```diff
@@ -169,6 +169,9 @@ service NetworkServerService {
 
     // GetADRAlgorithms returns the available ADR algorithms.
     rpc GetADRAlgorithms(google.protobuf.Empty) returns (GetADRAlgorithmsResponse) {}
+
+    // ClearDeviceDevNonce clear device devnonce from activation records.
+    rpc ClearDeviceDevNonce(ClearDeviceDevNonceRequest) returns (google.protobuf.Empty) {}
```

Comment:
I think I would call this `ClearDeviceNonces` / `ClearDeviceNoncesRequest` as it clears multiple items, not a single entry.

Reply:
I have made changes as per your comments.

I will make other PRs ASAP